packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -93,7 +97,7 @@ const propTypes = {
 
 const defaultProps = {
   rowHeaderIndex: 0,
-  columnWidth: '200px',
+  defaultColumnWidth: 200,
```

Comment:
On second thought, this seems to be removing the ability for consumers to specify units. Is there a reason we have to use a number instead of parsing the string internally?

Reply:
We are working with pixels so that we have the ability to resize columns.  It is the same as clinical data grid.